src/components/BaseCard.vue

```diff
@@ -1,76 +1,82 @@
 <template>
-  <div
-    class="flex flex-col overflow-hidden rounded-lg shadow-md hover:shadow-xl transition-shadow"
+  <router-link
+      :to="getId"
```

Comment:
getId name i a bit missleading, better naming would be blogRoute or blogIdRoute etc